.github/workflows/release.yaml

```diff
@@ -17,10 +17,10 @@ jobs:
         with:
           token: ${{ secrets.VIB_ACTION_TOKEN }}
 
-      - name: Set Node.js 16.x
+      - name: Set Node.js 20
         uses: actions/setup-node@v2
         with:
-          node-version: 16.x
+          node-version: '20'
```

Comment:
Why `'20'` instead of `20.x`. I guess they are similar, but I'm not sure. Also, there is no need to add `'`.